app/Routes.php

```diff
@@ -66,20 +66,15 @@ public static function initializeRoutes( Application $app, FunFunFactory $ffFact
 			'validate-donation-amount',
 			function ( Request $httpRequest ) use ( $app, $ffFactory ) {
 
-				$constraint = new Collection(
-					[
-						'allowExtraFields' => false,
-						'fields' => [
-							'amount' => $ffFactory->newDonationAmountConstraint()
-						]
-					]
-				);
+				$rawAmount = $httpRequest->request->get('amount','');
 
-				$violations = Validation::createValidator()->validate( $httpRequest->request->all(), $constraint );
+				if(!ctype_digit($rawAmount)){
```

Comment:
This might trigger the code style check - we usually put spaces after opening brackets.